docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-buster
+FROM python:3.9-slim-buster
```

Comment:
As long as our production environment is using Python 3.7, I think our local ones should run the same.

Reply:
You're right